### PR TITLE
Wrong selector expression, unable to move parent category

### LINKED ISSUE #15344: 1.7.6.1 Moving parent category to sub category not possible. Not expanding - Safari browser
Im trying to move a category to a sub category but sub categorys are not expanding. Not able to move.
>[Edited by Kholoud]
 We need to use Safari browser (Chrome => OK)

PrestaShop version: 1.7.6.1
PHP version: 7.2

### PR DESCRIPTION
**Pull request by PierreRambaud**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 1.7.6.x
| Description?  | The css expression is currently broken for the `defaultTitle`.
| Type?         | bug fix
| Category?     | BO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | Fixes #15344 
| How to test?  | You shouldn't have error on Safari and should be able to move parent category.

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/15665)

### CODE REVIEW

**Review by matthieu-rolland — DISMISSED** [review]
lgtm

File: admin-dev/themes/new-theme/js/pages/category/index.js [hunk 1]
```diff
@@ -89,7 +89,7 @@ $(() => {
   new Serp(
     {
       container: '#serp-app',
-      defaultTitle: 'input[name^="category[name]',
+      defaultTitle: 'input[name^="category[name]"]',
       watchedTitle: 'input[name^="category[meta_title]',
```
Comment (Progi1984) [inline admin-dev/themes/new-theme/js/pages/category/index.js]: @PierreRambaud Is it normal that another css selectors has not the ending quote & square bracket ?

Comment (PierreRambaud) [reply admin-dev/themes/new-theme/js/pages/category/index.js]: Oh no, you're right, I'll update this

**Review by Progi1984 — APPROVED** [review]

**Comment by PierreRambaud** [comment]
Pull request still in queue but everything is ok https://travis-ci.com/PrestaShop/PrestaShop/builds/128877882 :thinking:

**Comment by jolelievre** [comment]
I closed/reopened it to relaunch the build, let's wait for it!

**Review by jolelievre — APPROVED** [review]

**Review by toutantic — APPROVED** [review]

**Review by matks — APPROVED** [review]

**Pull request merged**